domains/maherin.json

```
@@ -0,0 +1,11 @@
+{
+    "description": "Describe the use of this subdomain",
+    "repo": "https://github.com/MaherinShaikh/Personal-Portfolio",
+    "owner": {
+        "username": "MaherinShaikh",
+        "email": "maherin2114@gmail.com"
+    },
+    "record": {
+        "CNAME": "MaherinShaikh.github.io"
```

incomplete [website](https://maherinshaikh.github.io/Personal-Portfolio/), not much content and everything else gives 404
```suggestion
        "CNAME": "maherinshaikh.github.io"
```